tests/test_bypassing_validation.py

```diff
@@ -0,0 +1,65 @@
+import inspect
+
+import pytest
+from pydantic import ValidationError
+
+from bo4e.bo.marktlokation import Marktlokation
+from bo4e.com.adresse import Adresse
+from bo4e.enum.bilanzierungsmethode import Bilanzierungsmethode
+from bo4e.enum.energierichtung import Energierichtung
+from bo4e.enum.sparte import Sparte
+
+
+class TestValidationBypass:
+    """
+    test cases that shows how to instantiate objects even without required fields
+    """
+
+    def test_instantiation_with_missing_required_attributes(self) -> None:
+        # Assuming you'd like to instantiate a Marktlokation but have no value for e.g. the netzebene.
+        # Then you could try the usual constructor:
+        def instantiate_with_constructor() -> Marktlokation:
+            malo = Marktlokation(  # type:ignore[call-arg] # silence mypy complaints about the missing netzebene
+                marktlokations_id="51238696781",
+                sparte=Sparte.GAS,
+                lokationsadresse=Adresse(
+                    postleitzahl="04109", ort="Leipzig", hausnummer="15", strasse="Thomaskirchhof"
+                ),
+                energierichtung=Energierichtung.EINSP,
+                bilanzierungsmethode=Bilanzierungsmethode.PAUSCHAL,
+                # note that netzebene is required but not given!
+                # netzebene=Netzebene.NSP,
+            )
+            return malo
+
+        # now, instantiation will fail:
+        with pytest.raises(ValidationError) as validation_error:
+            instantiate_with_constructor()
+        error_msg = str(validation_error.value).replace("\n", " ").replace("\r", " ").replace("  ", " ")
```

Comment:
das ist nicht schön aber macht den punkt klar, wenn man es in der nächsten zeile vergleicht.

Reply:
Warum nicht einfach
```python
assert "1 validation error for Marktlokation netzebene" in validation_error.value
assert "field required" in validation_error.value
```
?

Reply:
es bockt wegen der linebreaks.
![grafik](https://user-images.githubusercontent.com/23094997/205149142-367d4b40-1ff5-4d83-8abf-53d2e7c70159.png)